src/main/scala/scorex/core/network/peer/PeerInfo.scala

```diff
@@ -2,19 +2,29 @@ package scorex.core.network.peer
 
 import java.net.InetSocketAddress
 
-import scorex.core.network.{ConnectionType, PeerFeature}
+import scorex.core.app.Version
+import scorex.core.network.{ConnectionType, Handshake, PeerData}
 
-case class PeerInfo(lastSeen: Long,
-                    declaredAddress: Option[InetSocketAddress],
-                    nodeName: Option[String] = None,
-                    connectionType: Option[ConnectionType] = None,
-                    features: Seq[PeerFeature] = Seq()) {
+/**
+  * Information about peer to be stored in PeerDatabase
+  *
+  * @param peerData       - general information about the peer
+  * @param lastSeen       - timestamp when this peer was last seen in the network
+  * @param connectionType - type of connection (Incoming/Outgoing) established to this peer if any
+  */
+case class PeerInfo(peerData: PeerData,
```

Comment:
`PeerInfo` is just stored information about peer, it is not necessary connected.

Reply:
@catena2w, I mean, that it would be better to make it obvious that the difference between `PeerData` and `PeerInfo` is that in the latter the information about peer appears in the context of connection.